setup.py

```diff
@@ -51,7 +51,7 @@ def default_setup_args(*, version):
         name=name,
         version=version,
         author="AutoGluon Community",
-        url="https://github.com/autogluon/autogluon-cloud",
+        url="https://github.com/autogluon/autogluon-fair",
         description="Train and deploy AutoGluon backed models on the cloud",
```

Comment:
This needs to be updated too

Reply:
This was addressed in the other PR, will update the other PR so I can get this fixed here